### ISSUE TITLE
Fix warning RemovedInDjango41Warning

### ISSUE THREAD

**Issue by gitenth** [burke-software/django-mass-edit#108]
Please fix warning:

`RemovedInDjango41Warning: 'massadmin' defines default_app_config = 'massadmin.apps.MassAdminConfig'. Django now detects this configuration automatically. You can remove default_app_config.
    app_config = AppConfig.create(entry)
`

add code into `massadmin/__init__.py`:

```

import django

if django.VERSION < (3, 2):
    default_app_config = 'massadmin.apps.MassAdminConfig'
```

**Comment by PetrDlouhy** [comment]
@gitenth Could you please convert your comment into Pull request?

**Comment by PetrDlouhy** [comment]
@gitenth I have released `django-mass-edit` version 3.5.0 which should support Django up to 4.2.
Is this issue still a problem in that version?

**Comment by samuller** [comment]
Yes, it looks like it's still an issue with v3.5.0. I see the warning when I run Django with warnings enabled:

    PYTHONWARNINGS=default python manage.py runserver
